ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.j2

```diff
@@ -70,6 +70,137 @@
   # Set the user agent for the POSTs (can be useful for log differentiation)
   # user_agent = "telegraf"
 
+  [outputs.influxdb.tagdrop]
+    app = ["mozlocal*", "wonk", "joe-fox", "pdfit", "tilex", "kairos", "seomoz", "leeloo", "longboard", "inbound"]
+
+[[outputs.influxdb]]
```

Comment:
it seems inefficient and inelegant to hard code this configuration when it could be dynamically created with jinja.